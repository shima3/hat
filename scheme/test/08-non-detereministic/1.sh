#!/bin/bash
$* -e main examples/non-deterministic.sch
