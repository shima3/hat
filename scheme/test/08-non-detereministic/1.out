begin
3124
end
